Patch-Source: https://gitlab.freedesktop.org/pulseaudio/pavucontrol/-/commit/964f2987a46aa984d9ba83369b1f27c0bb28e9af
--
From 964f2987a46aa984d9ba83369b1f27c0bb28e9af Mon Sep 17 00:00:00 2001
From: Arun Raghavan <arun@asymptotic.io>
Date: Sat, 15 Jan 2022 13:45:00 -0500
Subject: [PATCH] Fix warning in .desktop file

With the underscore, we got:

../src/pavucontrol.desktop.in:12:0: warning: invalid non-blank line
---
 src/pavucontrol.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pavucontrol.desktop.in b/src/pavucontrol.desktop.in
index 7499e2e..0298219 100644
--- a/src/pavucontrol.desktop.in
+++ b/src/pavucontrol.desktop.in
@@ -8,4 +8,4 @@ Icon=multimedia-volume-control
 StartupNotify=true
 Type=Application
 Categories=AudioVideo;Audio;Mixer;GTK;Settings;X-XFCE-SettingsDialog;X-XFCE-HardwareSettings;
-_Keywords=pavucontrol;Microphone;Volume;Fade;Balance;Headset;Speakers;Headphones;Audio;Mixer;Output;Input;Devices;Playback;Recording;System Sounds;Sound Card;Settings;Preferences;
+Keywords=pavucontrol;Microphone;Volume;Fade;Balance;Headset;Speakers;Headphones;Audio;Mixer;Output;Input;Devices;Playback;Recording;System Sounds;Sound Card;Settings;Preferences;
-- 
GitLab

